helix-term/src/keymap.rs

```diff
@@ -309,13 +318,23 @@ pub fn default() -> Keymaps {
 
             key!('t') => commands::extend_till_char,
             key!('f') => commands::extend_next_char,
+
             key!('T') => commands::extend_till_prev_char,
             key!('F') => commands::extend_prev_char,
-
+            key!('^') => commands::extend_line_start,
+            KeyEvent {
+                code: KeyCode::Home,
+                modifiers: KeyModifiers::NONE
+            } => commands::extend_line_start,
+            key!('$') => commands::extend_line_end,
+            KeyEvent {
+                code: KeyCode::End,
+                modifiers: KeyModifiers::NONE
+            } => commands::extend_line_end,
```

Comment:
Can you just merge them with `|`?

Reply:
No point since they're deleted now, but yes, why not; I just wanted to be consistent with the rest of the code.